src/mlpack/core/optimizers/adam/adam.hpp

```diff
@@ -5,10 +5,10 @@
  * @author Marcus Edel
  * @author Vivek Pal
  *
- * Adam and AdaMax optimizer. Adam is an an algorithm for first-order gradient-
+ * Adam, AdaMax and AMSGrad optimizer. Adam is an an algorithm for first-order gradient-
```

Comment:
Lines should be no more than 80 characters wide; for more information take a look at: https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping, the same applies to line 13 and 56.

Reply:
I skipped that while editing. I have made edits to those lines.